packages/psss/src/api/queries/__tests/useCountryWeeklyReport.test.js

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
There is another card for writing unit tests and so more detail and coverage will be added later.